lib/base-compiler.ts

```diff
@@ -431,7 +431,7 @@ export class BaseCompiler implements ICompiler {
         const key = this.getCompilerCacheKey(compiler, args, optionsForCache);
         let result = await this.env.compilerCacheGet(key as any);
         if (!result) {
-            result = await this.env.enqueue(async () => await this.exec(compiler, args, options));
+            result = await this.env.enqueue(async () => await this.exec(compiler, args, <ExecutionOptions>options));
```

Comment:
It'd be best to `unwrap()` here. Even though it's definitely not undefined we've been using `unwrap` as a checked non-null/undefined assertion so we can get better diagnostics if things ever go wrong in the future.